yesod-core/Yesod/Core/Internal/Session.hs

```diff
@@ -63,7 +61,7 @@ clientSessionDateCacher validity = do
       , updateFreq   = 10000000 -- 10s
       }
 
-    return $! (getClientSessionDateCache, return ())
```

Comment:
No, that should be removed. It's probably a relic of a refactor, forcing evaluation of a tuple like this never makes sense.